### PR TITLE
Clean loading results

### PR DESCRIPTION
**Pull request by jotoeri**
Now using formsService, as well as the Views-Mixin for storage.

### CODE REVIEW

File: src/views/Results.vue [hunk 1]
```diff
@@ -117,14 +117,12 @@ export default {
...
 			loadingResults: true,
-			submissions: [],
-			questions: [],
 		}
 	},
 
 	computed: {
 		noSubmissions() {
-			return this.submissions && this.submissions.length === 0
+			return this.form.submissions && this.form.submissions.length === 0
```
Comment (skjnldsv) [inline src/views/Results.vue:125]:
```suggestion
			return this.form.submissions?.length === 0
```

File: lib/Service/FormsService.php [hunk 2]
```diff
@@ -93,10 +93,10 @@ public function getOptions(int $questionId): array {
 		}
 	}
 
-	public function getQuestions(int $formId): array {
+	public function getQuestions(int $formId, bool $loadDeleted = false): array {
```
Comment (skjnldsv) [inline lib/Service/FormsService.php:96]: phpdocs? :p

Comment (jotoeri) [reply lib/Service/FormsService.php:96]: Haha, there were none before already! :D
But yes, can do that...

**Comment by jancborchardt** [comment]
Which bug does it fix, or how to test @jotoeri? :)

**Comment by jotoeri** [comment]
None, it's just a bit of code-cleanup in results.
So as long as results are still shown, deleted Questions are shown in Results if answered but not on edit, then it should work^^

**Comment by jotoeri** [comment]
Sorry. Didn't change anything here, just accidentially pushed a separate commit. It's corrected again.

**Comment by jotoeri** [comment]
Closing in favor of #386

**Pull request closed without merging**